### ISSUE TITLE
Tooltip grows bigger and bigger

### ISSUE THREAD

**Issue by Texxer** [antrix1/arenamaster-addon#22]
Happens randomly, but often. The info text gets duplicated over and over again, and the window grows bigger and bigger since Dragonflight. It doesn't throw any errors and resets on switching targets.

![Wow_9hAj5oTNEe](https://user-images.githubusercontent.com/22458550/209447481-4ca2b30b-f5c2-4428-bfc9-12f17a1fc15f.jpg)

**Comment by antrix1** [comment]
Thanks for the report. I noticed it as well. Trying to debug it. Have you noticed any pattern when it occurs? For me, I believe it happened in arena when mousing over an enemy nameplate.

**Comment by CreepyJoker** [comment]
I've updated a few bits now, tho I'm not sure if it will fix the issue as I was not able to reproduce it on my end. The mouseover on nameplates doesn't trigger it, mouseover on raid/party frames don't either. I've tested with some other addons enabled that hook the tooltip as well, but no luck on my end.

I'll keep the issue open, let me know if with the next update it still does the same ( antrix has to push a new release for today ).

**Comment by Texxer** [comment]
It doesn't happen as often anymore but it still occurs frequently. Sometimes, it also pops up when I mouse over walls or towers in Epic Battlegrounds. I am using ElvUI together with the Benik UI mod.

**Comment by Texxer** [comment]
It got much worse since the last update. It fills the whole screen now and crashes my game.

**Comment by CreepyJoker** [comment]
Hmm, very strange. I'll take a look at it again tomorrow morning and rewrite the whole tooltip logics. It clearly doesn't like something since dragonflight.

**Comment by Texxer** [comment]
Yeah, it's hard to debug when it doesn't throw an error. It doesn't bug out in the open world if that helps.

**Comment by CreepyJoker** [comment]
Hopefully with the next data update it gets solved (tomorrow it's the next one). I've tested it in like 10 bgs with RaiderIO enabled, ElvUI and many other addons that hook the gametooltip and the problem doesn't seem to be occuring anymore. Before that I've managed to reproduce it a few times after installing ElvUI.

If there was to be even an error popping, it would've helped a lot. Keep in mind that also the BlizzardUI has problems that aren't fixed by now and taints it's UI itself. That can also be a cause for many "unexpected / error-less" issues. It has mainly issues on the EditMode / hooks towards that and cause issues with other addons.

**Comment by Texxer** [comment]
Nice, it seems you were able to fix it!

**Comment by Texxer** [comment]
Hmm, the bug happened again.

**Comment by Texxer** [comment]
`69x Texture:SetTextCoord(): TexCoord out of range
[string "=[C]"]: ?
[string "=[C]"]: in function `SetTexCoord'
[string "@SharedXML/Backdrop.lua"]:208: in function <SharedXML/Backdrop.lua:207>
[string "@SharedXML/Backdrop.lua"]:240: in function `SetupTextureCoordinates'
[string "@SharedXML/Backdrop.lua"]:184: in function <SharedXML/Backdrop.lua:182>
[string "=[C]"]: in function `GetWidth'
[string "@SharedXML/Backdrop.lua"]:215: in function `SetupTextureCoordinates'
[string "@SharedXML/Backdrop.lua"]:325: in function `ApplyBackdrop'
[string "@SharedXML/Backdrop.lua"]:341: in function `SetBackdrop'
[string "@ElvUI/Core/General/Toolkit.lua"]:163: in function `SetTemplate'
[string "@ElvUI/Core/Modules/UnitFrames/Elements/InfoPanel.lua"]:42: in function <...ElvUI/Core/Modules/UnitFrames/Elements/InfoPanel.lua:16>
[string "=[C]"]: in function `Configure_InfoPanel'
...
[string "@ElvUI/Core/Modules/UnitFrames/UnitFrames.lua"]:1015: in function `CreateAndUpdateHeaderGroup'
[string "@ElvUI/Core/Modules/UnitFrames/UnitFrames.lua"]:1112: in function `UpdateAllHeaders'
[string "@ElvUI/Core/Modules/UnitFrames/UnitFrames.lua"]:617: in function <ElvUI/Core/Modules/UnitFrames/UnitFrames.lua:577>
[string "=[C]"]: in function `Update_AllFrames'
[string "@ElvUI/Core/Modules/UnitFrames/UnitFrames.lua"]:892: in function `?'
[string "@BattleGroundEnemies/Libs/CallbackHandler-1.0-7/CallbackHandler-1.0.lua"]:119: in function <...ies/Libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:119>
[string "=[C]"]: ?
[string "@BattleGroundEnemies/Libs/CallbackHandler-1.0-7/CallbackHandler-1.0.lua"]:29: in function <...ies/Libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:25>
[string "@BattleGroundEnemies/Libs/CallbackHandler-1.0-7/CallbackHandler-1.0.lua"]:64: in function `Fire'
[string "@ConfirmSP/Libs/AceEvent-3.0-4/AceEvent-3.0.lua"]:120: in function <ConfirmSP/Libs/AceEvent-3.0/AceEvent-3.0.lua:119>

Locals:
(*temporary) = Texture {
 0 = <userdata>
 PixelSnapDisabled = true
}
(*temporary) = 0.132813
(*temporary) = 0.062500
(*temporary) = 0.132813
(*temporary) = 4785003260.076438
(*temporary) = 0.242188
(*temporary) = 0.062500
(*temporary) = 0.242188
(*temporary) = 4785003260.076438
`

**Comment by QuinnGT** [comment]
Glad to find this thread. It happens to me when I go into any pvp match solos, rated, random, etc. For now I've just disabled the loading of it on those. Only use it in world. I'll get you an error log. (not using elvui or raider io)

**Comment by Texxer** [comment]
Can we start working on this one, please? @antrix1

**Comment by nonstopjoker** [comment]
This is happening because of the changes on https://github.com/antrix1/arenamaster-addon/commit/4b9bdc0168a734d5a8d3d6eac78da7de07cf842e
